src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -266,14 +266,14 @@ class Browser extends DashboardView {
     });
   }
 
-  fetchAggregationPanelData(objectId, className) {
+  fetchAggregationPanelData(objectId, className, appID) {
```

Comment:
```suggestion
  fetchAggregationPanelData(objectId, className, appId) {
```

Use camelCase for var names